README.md

```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
 ./develop_server stop
 ```
 
+## Heroku Review Apps ##
+
+Heroku is a deployment service we use to run test versions of our app. Heroku
+can make a unique URL for each open pull request, where you can view a
+temporary version of the website with the changes from that pull request.
+
+### Permissions to access Heroku
+
+Access to Heroku isn't granted to everyone, but at least someone currently
+involved in the WiCS committee will ideally have the permissions. You don't
```

Comment:
This should probably reflect how we actually hand out access, which is

> Access to Heroku is granted to all committers in the wics-uw GitHub organization. Membership to the "committers" group is granted to all full Systems Committee members, as well as individuals that the Systems Committee have chosen in recognition of their contributions and demonstrated responsibility and good judgment.
>
> To become a committer, you'll need to learn git, contribute ... , and then ask the Systems Committee for access, by emailing `<wics-sys@lists.uwaterloo.ca>`.

^ I added the backticks there to show how to add an email in Markdown.